packages/thirdweb/src/v2/transactions/get.ts

```diff
@@ -0,0 +1,70 @@
+import { getTransactionById } from "@thirdweb-dev/api";
+import type { ThirdwebClient } from "../../client/client.js";
+import { getThirdwebBaseUrl } from "../../utils/domains.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import type { Transaction } from "./types.js";
+import type { Address } from "../../utils/address.js";
+import type { Hex } from "../../utils/encoding/hex.js";
+
+/**
+ * Retrieves a transaction by ID.
+ *
+ * @param options - Options including the transaction ID
+ * @param options.transactionId - The ID of the transaction to retrieve
+ * @returns Promise that resolves to the transaction
+ * @example
+ *
+ * ## Get a transaction by ID
+ * ```typescript
+ * import { Client, Transactions } from "thirdweb/v2";
```

Comment:
thinking more and more that we can get away with just import from "thirdweb" with the new namespaces. Just so much simpler. over time we can deprecate/move things out of the main export